packages/@okta/vuepress-site/docs/reference/api/threat-insight/index.md

```diff
@@ -5,17 +5,18 @@ category: management
 
-In order to prevent abuse, Okta ThreatInsight works in a limited capacity for free trial editions. Please contact Okta support if fully functional Okta ThreatInsight is required.
+The Okta ThreatInsight Configuration API provides operations to manage your ThreatInsight configuration.
 
-## ThreatInsight configuration object
+> **Note:** In order to prevent abuse, Okta ThreatInsight works in a limited capacity for free trial edition orgs. Please contact Okta support if fully functional Okta ThreatInsight is required.
 
-| Field Name     | Description                                                         	| Data Type                                     | Required      | Max Length    |
-| :------------- | :------------------------------------------------------------------	| :-------------------------------------------- | :------------ | :------------ |
-| action         | Specifies how Okta responds to authentication requests from suspicious IPs. Values are none, audit, or block. A value of none indicates that ThreatInsight is disabled. A value of audit indicates that Okta logs suspicious requests in the System Log. A value of block indicates that Okta logs suspicious requests in the System Log and blocks the requests. | String                                        | Yes		| N/A           |
-| excludeZones   | Accepts a list of [Network Zone](/docs/reference/api/zones/) IDs. IPs in the excluded Network Zones aren't logged or blocked by Okta ThreatInsight and proceed to Sign On rules evaluation. This ensures that traffic from known, trusted IPs isn't accidentally logged or blocked. | List	                                        | No		| N/A           |
+## ThreatInsight configuration object
 
+| Field Name  | Description	| Data Type   | Required      |
+| :---------- | :----------	| :---------- | :------------ |
+| action         | Specifies how Okta responds to authentication requests from suspicious IPs. Supported values:<br><ul><li>`none`: Indicates that ThreatInsight is disabled</li><li>`audit`: Indicates that Okta logs suspicious requests in the System Log</li><li>`block`:  Indicates that Okta logs suspicious requests in the System Log and blocks the requests</li></ul> | String (enums: `none`, `audit`, or `block`) | Yes |
+| excludeZones   | Accepts a list of [Network Zone](/docs/reference/api/zones/) IDs. IPs in the excluded network zones aren't logged or blocked and requests proceed to authentication rules evaluation. This ensures that traffic from known, trusted IPs isn't accidentally logged or blocked. | List | No |
```

Comment:
- suggest add a comma after "logged or blocked,"
- 
-

Reply:
Thanks. I'm going to remove the "requests proceed to..." part. It doesn't add value.